src/Main.cc

```diff
@@ -491,13 +491,12 @@ static void showSplash()
                  ord::OpenRoad::getGitDescribe());
   logger->report(
       "Features included (+) or not (-): "
-      "{}Charts {}GPU {}GUI {}Python{}",
+      "{}Charts {}GPU {}GUI {}Python {}",
       ord::OpenRoad::getChartsCompileOption() ? "+" : "-",
       ord::OpenRoad::getGPUCompileOption() ? "+" : "-",
       ord::OpenRoad::getGUICompileOption() ? "+" : "-",
       ord::OpenRoad::getPythonCompileOption() ? "+" : "-",
-      strcmp(BUILD_TYPE, "Release") == 0 ? ""
-                                         : fmt::format(" : {}", BUILD_TYPE));
```

Comment:
Printing out the buid type unconditionally is congruent with always printing out the feature above, even if it is not compiled in.

I wonder what the strcmp() was doing. In my case BUILD_TYPE == "RELEASE", which doesn't match the strcmp() above.